### PR TITLE
CSHARP-5057: Support Range Indexes as GA

### PR DESCRIPTION
**Pull request by BorisDog**

### CODE REVIEW

**Review by sanych-sun — APPROVED** [review]
LGTM

File: src/MongoDB.Driver/MongoDB.Driver.csproj [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
-    <PackageReference Include="MongoDB.Libmongocrypt" Version="1.11.0" />
+    <PackageReference Include="MongoDB.Libmongocrypt" Version="1.11.1.1" />
```
Comment (adelinowona) [inline src/MongoDB.Driver/MongoDB.Driver.csproj:23]: Update before merging

File: src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <PackageReference Include="AWSSDK.SecurityToken" Version="3.7.100.14" />
     <PackageReference Include="DnsClient" Version="1.6.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
-    <PackageReference Include="MongoDB.Libmongocrypt" Version="1.11.0" />
+    <PackageReference Include="MongoDB.Libmongocrypt" Version="1.11.1.1" />
```
Comment (adelinowona) [inline src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj:31]: Update before merging

File: src/MongoDB.Driver.Core/Core/Misc/Feature.cs [hunk 3]
```diff
@@ -376,6 +377,11 @@ public class Feature
         /// </summary>
         public static Feature Csfle2QEv2RangePreviewAlgorithm => __csfle2Qev2RangePreviewAlgorithm;
```
Comment (adelinowona) [inline src/MongoDB.Driver.Core/Core/Misc/Feature.cs:378]: Consider marking as Obselete

**Review by adelinowona — APPROVED** [review]
LGTM + some small comments

**Pull request merged**